api/package.json

```diff
@@ -3,8 +3,12 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www",
-    "queue": "node worker-start.js"
+    "start:api": "nodemon --watch 'api/vudl.ini' --watch 'api/**/*.js' --watch 'api/**/*.ejs' --ignore api/node_modules/ src/server.ts",
+    "start:queue": "nodemon --watch 'api/vudl.ini' --watch 'api/**/*.js' api/worker-start.ts",
+    "start:server": "ts-node src/server.ts",
+    "start:worker": "ts-node src/worker-start.ts",
```

Comment:
I'd still change this a little more -- isn't start:api the dev version of start:server, and start:queue the dev version of start:worker? If I'm understanding correctly, should we rename start:api --> start:serverDev and start;queue --> start:workerDev?

Reply:
One of them watches for chagnes in the code I think.

Reply:
True, so maybe renaming them with "Watch" instead of "Dev" would be more meaningful... start:serverWatch and start:workerWatch?

Reply:
Okay. so I'll rename it to that, but I do know the original behavior of dev was to watch api and queue also.

Reply:
The top-level dev task is concurrently running `npm:client npm:api:dev npm:queue:dev`. I think as long as we don't change the behavior of that one, giving the deeper tasks more meaningful names should be fine. It might be worth asking @crhallberg for input, though, since he did the original NPM setup and might have a better sense than I do of how things are intended to be broken down.